### PR TITLE
dashcore masternode RPC support

### PR DESCRIPTION
**Pull request by mayoreee**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ use dashcore_rpc::{Auth, Client, RpcApi};
 
 fn main() {
 
-    let rpc = Client::new("http://localhost:8332".to_string(),
+    let rpc = Client::new(&"http://127.0.0.1:19998".to_string(),
```
Comment (QuantumExplorer) [inline README.md:29]: Why the borrow? Also why switch out localhost?

File: client/src/client.rs [hunk 2]
```diff
@@ -1170,12 +1177,12 @@ pub struct Client {
...
 impl fmt::Debug for Client {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        write!(f, "bitcoincore_rpc::Client({:?})", self.client)
+        write!(f, "dashcore_rpc::Client({:?})", self.client)
     }
 }
 
 impl Client {
-    /// Creates a client to a bitcoind JSON-RPC server.
+    /// Creates a client to a dashcored JSON-RPC server.
```
Comment (QuantumExplorer) [inline client/src/client.rs:1185]: dashd

File: json/src/lib.rs [hunk 3]
```diff
@@ -8,27 +8,27 @@
 // If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.
 //
 
-//! # Rust Client for Bitcoin Core API
+//! # Rust Client for Dashcore Core API
```
Comment (QuantumExplorer) [inline json/src/lib.rs:11]: Dash Core

File: json/src/lib.rs [hunk 4]
```diff
@@ -8,27 +8,27 @@
...
 //!
-//! This is a client library for the Bitcoin Core JSON-RPC API.
+//! This is a client library for the Dashcore Core JSON-RPC API.
```
Comment (QuantumExplorer) [inline json/src/lib.rs:13]: Dash Core

**Review by QuantumExplorer — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (mayoreee) [reply README.md]: fixed - swaped localhost in for IP address

File: json/src/lib.rs (hunk 3 above)
Comment (mayoreee) [reply json/src/lib.rs]: fixed - updated name

File: client/src/client.rs (hunk 2 above)
Comment (mayoreee) [reply client/src/client.rs]: fixed - updated name

File: json/src/lib.rs (hunk 4 above)
Comment (mayoreee) [reply json/src/lib.rs]: fixed - updated name

**Review by QuantumExplorer — APPROVED** [review]

**Pull request merged**